server/tests/test_data/nodejs_query/scatter/screenshot.json

```diff
@@ -1,46 +1,3 @@
-      "srcs": [
-        {
-          "name": "census.gov",
-          "url": "https://www.census.gov/programs-surveys/acs/data/data-via-ftp.html"
-        }
-      ],
-      "title": "Population Below Poverty Line in Counties of California (2021)",
-      "type": "MAP",
-      "unit": "",
-      "chartUrl": ""
-    }
-  ]
+  "charts": []
```

Comment:
Why don't we see the ranking chart that's 3rd in https://datacommons.org/explore/#q=obesity%20vs.%20poverty%20in%20counties%20of%20california ?